libfdo-data-go/ownershipvoucher.go

```diff
@@ -0,0 +1,86 @@
+package libfdo_data
+
+// #include <libfdo-data/fdo_data.h>
+import "C"
+
+import (
+	"fmt"
+)
+
+type OwnershipVoucherList struct {
+	inner *C.struct_FdoOwnershipVoucherList
+}
+
+func ParseManyOwnershipVouchers(voucherB []byte) (*OwnershipVoucherList, error) {
+	ctslen := C.size_t(len(voucherB))
+	cts := C.CBytes(voucherB)
+
+	voucherlist := C.fdo_ownershipvoucher_many_from_data(cts, ctslen)
+	if voucherlist == nil {
+		return nil, fmt.Errorf("failed to parse ownership vouchers: %v", getLastError())
+	}
+
+	return &OwnershipVoucherList{
+		inner: voucherlist,
+	}, nil
+}
+
+func (ovl *OwnershipVoucherList) Free() {
+	C.fdo_ownershipvoucher_list_free(ovl.inner)
+}
+
+func (ovl *OwnershipVoucherList) Len() int {
+	return int(C.fdo_ownershipvoucher_list_len(ovl.inner))
+}
+
+func (ovl *OwnershipVoucherList) GetVoucher(index int) *OwnershipVoucher {
```

Comment:
I would do this as:

```go
func (ovl *OwnershipVoucherList) GetVoucher(index int) (*OwnershipVoucher, error) {
```

err in golang is always checked anyway and you can return a Not Found in the `== nil` case